src/main/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -704,6 +704,8 @@ private void generateDuplicateIndexes(final boolean useBarcodes, final boolean i
                     if (TAG_DUPLICATE_SET_MEMBERS) {
                         addRepresentativeReadIndex(nextChunk);
                     }
+                } else if (nextChunk.size() == 1) {
+                    AbstractMarkDuplicatesCommandLineProgram.addSingletonToCount(libraryIdGenerator);
```

Comment:
It looks like this method is in scope (since this class is a subclass of `AbstractMarkDuplicatesCommandLineProgram`. So the prefix `AbstractMarkDuplicatesCommandLineProgram.` is not needed here or on line 721 below.